service/src/service_utils/asynchronous.py

```
@@ -324,10 +347,17 @@ def call_handlers_tuple(self, tuple):
+                # the client connection map, in such case tries to
+                # close the socket directly to avoid leaking file
+                # descriptors for pending or partially registered sockets
                 client_connection = self.client_connection_map.get(socket, None)
-                client_connection and client_connection.close()
+                if client_connection:
+                    client_connection.close()
+                else:
+                    try:
+                        socket.close()
+                    except Exception:
+                        pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)